acra/configuring-maintaining/logging/_index.md

```diff
@@ -1,4 +1,42 @@
 ---
-title: logging
+
+weight: 1
+title: Logging
 bookCollapseSection: true
 ---
+
+# Logging in Acra
+
+Starting with version [`0.77.0`](https://github.com/cossacklabs/acra/releases/tag/0.77.0), Acra supports [multiple logging formats](formats/):
+
+- plaintext
+- cef
+- json
```

Comment:
I believe these names should be properly capitalized here: CEF, JSON. These aren't command-line parameters.